test/core/dispatch/test_core.yaml

```diff
@@ -0,0 +1,12 @@
+
+#
+# Set up the pipeline for a 2-wide machine
+#
+#top.cpu:
+#  dispatch.num_to_dispatch: 2
+
+top.extension.core_extensions:
+  execution_topology:
+    [["alu", "1"],
+     ["fpu", "1"],
+     ["br",  "1"]]
```

Comment:
Suggest adding more pipes just like big_core.yaml or medium_core.yaml to test that scenarios

Reply:
Oh, definitely.  This is step 1 to allow for a dispatch tester I can use to do exactly that.  Step 2 in on a branch that I'm working on now.